lib/modules/manager/maven/index.spec.ts

```diff
@@ -229,7 +229,7 @@ describe('modules/manager/maven/index', () => {
       expect(packages).toMatchSnapshot();
     });
 
-    it('should not touch content if new and old versions are equal', () => {
+    it('should not edit content if new and old versions are equal', () => {
```

Comment:
This is a bit of a dubious change, I think _edit_ is the correct verb here. But I'm not familiar with the code, so please review this line. 😉